runtime/near-vm-logic/Cargo.toml

```diff
@@ -17,10 +17,13 @@ base64 = "0.13"
 borsh = "0.8.1"
 bs58 = "0.4"
 byteorder = "1.2"
+num-integer = "0.1"
```

Comment:
IMHO importing a library just to get ceiling division is an overkill.

Reply:
Fixed in https://github.com/near/nearcore/pull/4380/commits/4ceab9927161613027b27789100e2a5d29a87112